cypress/commons/utils/apiUtils.js

```diff
@@ -96,6 +96,20 @@ const interceptCreateScenario = () => {
   return alias;
 };
 
+const interceptUpdateScenario = (scenarioId) => {
+  const alias = forgeAlias('reqUpdateScenario');
+  cy.intercept({ method: 'PATCH', url: API_REGEX.SCENARIO, times: 1 }, (req) => {
+    const scenarioPatch = req.body;
+    if (stub.isEnabledFor('UPDATE_SCENARIO')) {
+      req.reply(scenarioPatch);
+      if (stub.isEnabledFor('GET_SCENARIOS')) stub.patchScenario(scenarioId, scenarioPatch);
+    } else if (stub.isEnabledFor('GET_SCENARIOS')) {
+      req.continue((res) => stub.patchScenario(scenarioId, scenarioPatch));
+    }
```

Comment:
It looks like `if (stub.isEnabledFor('GET_SCENARIOS')) stub.patchScenario(scenarioId, scenarioPatch);` could be factorized, and the `else if` be removed (there is no reason to call `req.continue`, you can let cypress forward the request to other interceptions or to the server if this was the last interception)